README.md

```diff
@@ -31,3 +31,19 @@ Used to standardize conventions across external interfaces.
 | HelperHeading       | DesiredHeading, GPS, HelperAISShip |
 | HelperLatLon        | GlobalPath, GPS, HelperAISShip     |
 | HelperSpeed         | GPS, HelperAISShip, WindSensor     |
+
+## Existing Libraries
```

Comment:
Can you link to the issue where we discuss this? There's some things in there that I could see being a useful reference in the future

Reply:
https://github.com/UBCSailbot/custom_interfaces/issues/3#issuecomment-1626875658

Reply:
I meant in the README lol

Reply:
🤦‍♂️ oops